src/Components/NavBarTest/Menus/MoreNavMenu.tsx

```diff
@@ -0,0 +1,34 @@
+import { Menu, MenuItem } from "@artsy/palette"
+import { AnalyticsSchema } from "Artsy"
+import { useTracking } from "Artsy/Analytics/useTracking"
+import React from "react"
+
+export const MoreNavMenu: React.FC<{ width?: number }> = ({ width = 160 }) => {
+  const { trackEvent } = useTracking()
+
+  const trackClick = event => {
+    const link = event.target
+    const text = link.innerText
+    const href = link.parentNode.parentNode.getAttribute("href")
+
+    trackEvent({
+      action_type: AnalyticsSchema.ActionType.Click,
+      context_module: AnalyticsSchema.ContextModule.HeaderMoreDropdown,
+      subject: text,
+      destination_path: href,
+    })
+  }
+
+  return (
+    <Menu onClick={trackClick} width={width}>
+      {/*
+        Hide nav items at md / lg as they appear in the top nav
+      */}
```

Comment:
This comment is a bit of an orphan now, since we don't do the tricky responsive props anymore in this iteration of the menu.